src/cmd/ksh93/tests/_common

```diff
@@ -19,7 +19,12 @@ _message()
 {
 	print -r $'\t'"${Command}[$1]: ${@:2}" >&2
 }
-alias err_exit='_message "$((Errors++,LINENO))"'   # inaccurate err_exit name kept for historical integrity :)
+function err_exit
+{
+	print -r $'\t'"${Command}[$1]: ${@:2}" >&2
```

Comment:
Can we change this to a `_message` call just to avoid having the formatting coded in two places instead of one? A single formatter helps ensure consistency.